cmd/web.go

```diff
@@ -88,7 +88,7 @@ func runWeb(ctx *cli.Context) error {
 	}
 	defer func() {
 		if panicked := recover(); panicked != nil {
-			log.Fatal("PANIC: %v\n%s", panicked, string(log.Stack(2)))
+			log.Fatal("panic occurred, no recover, PANIC: %v\n%s", panicked, log.Stack(2))
```

Comment:
Hmm I don't like this change of log line as it's pretty clear that it doesn't get recovered because it's logged at Fatal level. I don't understand what this "panic occurred, no recover," adds except make it more difficult to see the actual error that caused the panic.

```suggestion
			log.Fatal("PANIC: %v\n%s", panicked, log.Stack(2))
```